internal/mode/static/telemetry/collector_test.go

```diff
@@ -39,6 +39,24 @@ func createListCallsFunc(nodes []v1.Node) func(
 	}
 }
 
+func createGetCallsFunc(ns *v1.Namespace) func(
+	ctx context.Context,
+	nsNames types.NamespacedName,
+	obj client.Object,
+	opts ...client.GetOption,
+) error {
+	return func(ctx context.Context, nsNames types.NamespacedName, obj client.Object, opts ...client.GetOption) error {
+		switch obj.(type) {
+		case *v1.Namespace:
+			obj = ns
```

Comment:
I think this needs to be something like 

```
obj.ObjectMeta = ns.ObjectMeta
```
for the test to work.

I'm about to refactor the `createGetCallsFunc` in my PR so keep an eye on that as this might change a bit.

Reply:
`obj` doesn't have a ObjectMeta field